app/views/plugins/fundraisers/_fundraiser.liquid

```diff
@@ -66,7 +72,18 @@
 
   <script type="text/javascript">
     $(document).ready(function(){
-      var myFundraiserBar = new window.FundraiserBar("{{ follow_up_url }}");
+      var myFundraiserBar = new window.FundraiserBar({
+        followUpUrl: "{{ follow_up_url }}",
+        currency: "EUR",
+        donationBands: {
```

Comment:
`DEFAULT_DONATION_BANDS` is the constant as a fallback unless `donationBands` doesn't get passed.